testng-core-api/src/main/java/org/testng/xml/XmlSuite.java

```diff
@@ -123,6 +124,9 @@ public String toString() {
   private String m_parentModule = "";
   private String m_guiceStage = "";
 
+  /** Represents a unique id for this suite. Can be used for uniquely identifying the xml suite. */
+  public final UUID SUITE_ID = UUID.randomUUID();
```

Comment:
As not static, in camelCase

Reply:
I wanted this to stand out as if it is a constant. That was why I intentionally broke the naming conventions and used all CAPS